integration-test/utils/datafactory/schemas/footer.schema.ts

```diff
@@ -0,0 +1,91 @@
+export const footerSchema = {
+    "type": "object",
+    "properties": {
+      "title": {
+        "type": "string",
+        "minLength": 1
+      },
+      "subtitle": {
+        "type": "string",
+        "minLength": 1
+      },
+      "description": {
+        "type": "string",
+        "minLength": 1
+      },
+      "network": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "enum": ["linkedIn", "twitter", "github", "instagram", "email", "slack"]
+            },
+            "link": {
+              "type": "string", 
+              "minLength": 1
+            }
+          },
+          "required": ["type", "link"],
+          "additionalProperties": false,
+          "if": {
+            "properties": {
+              "type": { "const": "email" }
+            }
+          },
+          "then": {
+            "properties": {
+              "link": {
+                "pattern": "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"  // Custom email regex
+              }
+            }
+          },
+          "else": {
+            "properties": {
+              "link": {
+                "format": "uri",  // Format as URI
+                "pattern": "^https?://",  // URL pattern
+                "minLength": 1
+              }
+            }
+          }
+        },
+        "minItems": 6,
```

Comment:
Min can be 1 right?

Reply:
In this case I assumed that footer should contain links of all social media, that's why I put 6 for minItems and maxItems (means, it should be constant).